modules/homestakeros/default.nix

```diff
@@ -232,21 +234,34 @@ in {
                   BuilderProposals: true
 
               eth2:
-                BeaconNodeAddr: ws://${parsedConsensusEndpoint.addr}:${parsedConsensusEndpoint.port}:
+                BeaconNodeAddr: ${cfg.consensus.${consensusClient}.endpoint}
                 Network: mainnet
 
               eth1:
-                ETH1Addr: ${cfg.execution.${executionClient}.endpoint}
+                # This assumes that the websocket is bind to the same port, true for erigon, not for others
+                # TODO: Consider having a variable name for websocket endpoint
+                ETH1Addr: ws://${parsedExecutionEndpoint.addr}:8545
```

Comment:
I think that this should not be working. If I recall correctly, Erigon's `--ws.port` parameter did not exist before, and it seems to default to 8546 now, which is not the same as the HTTP-RPC port (8545) as I think it used to be (https://erigon.gitbook.io/erigon/advanced-usage/options).

Reply:
It seems like erigon does bind to both ports, but probably for compatibility reasons in older configurations. I updated the address to the new default, since the 8546 seems to be the one used by other clients as well.